src/main/java/com/mineboundteam/minebound/registry/BlockRegistry.java

```diff
@@ -22,12 +30,17 @@
 public class BlockRegistry {
     public static final DeferredRegister<Block> BLOCKS = DeferredRegister.create(ForgeRegistries.BLOCKS, MineBound.MOD_ID);
 
+    public static final RegistryObject<Block> ALLOY_FURNACE = BLOCKS.register("alloy_furnace", () -> new AlloyFurnace(BlockBehaviour.Properties.of(Material.STONE).requiresCorrectToolForDrops().strength(3.5F).lightLevel((b) -> b.getValue(BlockStateProperties.LIT) ? 15 : 0)));
```

Comment:
The alloy furnace gives off light all of the time. It also does not drop any of the contents inside of it when the block is broken.

Reply:
I'm not getting this, it's going out once smelting is done for me

Reply:
Looks like it goes out if smelting starts then finishes, but it is lit by default when it is placed down.

Reply:
Fixed